lib/peer_manager/include/modules/peer_manager_internal.h

```diff
@@ -128,14 +129,14 @@ typedef void (*pm_evt_handler_internal_t)(pm_evt_t *p_event);
  *
  * @return The number of words that the data takes in flash.
  */
-#define PM_BONDING_DATA_N_WORDS() BYTES_TO_WORDS(sizeof(pm_peer_data_bonding_t))
+#define PM_BONDING_DATA_N_WORDS() DIV_ROUND_UP(sizeof(pm_peer_data_bonding_t), sizeof(uint32_t))
```

Comment:
In that case, I would define `BYTES_TO_WORDS` here using `DIV_ROUND_UP` so that we don't repeat this code everywhere.